addons/website/migrations/11.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,72 @@
+---Fields in module 'website'---
+website      / ir.ui.view               / page (boolean)                : DEL
+#DONE: Copy ir.ui.view pages to website.page
+
+website      / website                  / auto_redirect_lang (boolean)  : NEW
+website      / website                  / compress_html (boolean)       : DEL
+website      / website                  / google_management_client_id (char): NEW
+website      / website                  / google_management_client_secret (char): NEW
+website      / website                  / homepage_id (many2one)        : NEW relation: website.page
+website      / website.menu             / page_id (many2one)            : NEW relation: website.page
+website      / website.page             / _inherits (False)             : NEW
+website      / website.page             / date_publish (datetime)       : NEW
+website      / website.page             / menu_ids (one2many)           : NEW relation: website.menu
+website      / website.page             / url (char)                    : NEW
+website      / website.page             / view_id (many2one)            : NEW relation: ir.ui.view, required: required
+website      / website.page             / website_ids (many2many)       : NEW relation: website
```

Comment:
This field should be filled on the SQL

Reply:
@pedrobaeza thank you, i made the changes

Reply:
@pedrobaeza could you check please